packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
```

Comment:
How about (I'm assuming it's the Okta console but not 100% sure of that):
``Create and configure SCIM integrations, and check the attributes with their corresponding mappings in the Okta console.``

Reply:
done